### PR TITLE
Add access to activation parameters for t4a and t4b tags. JB#50041

### PR DESCRIPTION
**Pull request by ikamaletdinov**
It's possible now to retrieve Activation parameters for ISO-DEP
Type 4B tags.

### CODE REVIEW

**Comment by monich** [comment]
Looks pretty straightforward but nfcd changes need to be merged first.

**Comment by ikamaletdinov** [comment]
Thanks for review, you are right, blocked by https://git.sailfishos.org/mer-core/nfcd/merge_requests/36

File: src/nci_adapter.c [hunk 1]
```diff
@@ -345,8 +358,17 @@ nci_adapter_create_known_tag(
...
-                nci_adapter_convert_poll_b(&poll_b, &mp->poll_b), NULL);
+            if (ntf->activation_param) {
+                const NciActivationParam* ap = ntf->activation_param;
+                NfcParamIsoDepPollB iso_dep_poll_b;
+
+                return nfc_adapter_add_tag_t4b(NFC_ADAPTER(self),
+                self->target, nci_adapter_convert_poll_b
+                    (&poll_b, &mp->poll_b),
+                nci_adapter_convert_iso_dep_poll_b
+                    (&iso_dep_poll_b, &ap->iso_dep_poll_b));
```
Comment (monich) [inline src/nci_adapter.c]: Indent please :)

Comment (ikamaletdinov) [reply src/nci_adapter.c]: Thanks, fixed

**Review by monich — APPROVED** [review]

**Pull request merged**